src/chapters/classes/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: Classes and Objects
```

Comment:
What do you think about adding one more exercise?  The students could instantiate the ``Student`` class and then print one or more of the properties.

Reply:
Added one! Let me know your thoughts!

Reply:
Looks good.